File: Programmers/자릿수 더하기.cpp
#include <iostream>

using namespace std;
int solution(int n)
{
    int answer = 0;
    while (n > 0)
    {
        answer += n % 10;
        n /= 10;
    }

    return answer;
}

int main() {
    int n;
    cin >> n;
    cout<<solution(n);
    system("pause");
}

File: Programmers/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        String s = String.valueOf(n);
        char [] c = s.toCharArray();
        
        for(int i = 0; i < c.length; i++)
            answer += c[i] - '0';
        return answer;
    }
}